mpmp/src/model/Card.java

```diff
@@ -17,53 +17,50 @@
 
 	public Card(String text, String code) {
 		this.text = text;
-
-		if (code.contains(";")) {
-			actions = code.split(";");
-		} else {
-			actions = new String[1];
-			actions[0] = code;
-		}
+		actions = code.split(";");
```

Comment:
If no `;` exists, the returned array has just one argument: the entire input string (here: `code`).